### ISSUE TITLE
clear the image cache except for downloaded songs

### ISSUE THREAD

**Issue by Spaceshiprace** [Malopieds/InnerTune#23]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

When clearing cache for images it clears also for the downloaded songs, It would be great if it clearing images except for those downloaded songs

### Why do you want this feature?

its annoying when it clearing for the songs that I downloaded, cuz everytime when I clear it I have to exit and reopen the app to load the images for the downloaded songs

### Additional information

I wish the description is clear enough ❤️

**Comment by fanieko** [comment]
Also, same images count for diferent songs count as 2 images, increasing image cache.

**Comment by Spaceshiprace** [comment]
yeah, I just realized that.. I wish it will be fixed also

**Comment by fanieko** [comment]
> yeah, I just realized that.. I wish it will be fixed also

yeah and it would be cool to upload custom images to fix normal video images to be 16:9 rectangles insteal of a 1:1 square

**Comment by Spaceshiprace** [comment]
I wish @Malopieds read these comments and work on it along with other enhancements that we mentioned ❤️🙏

**Comment by Malopieds** [comment]
Hi, of course I read every comments here! I don't have enough knowledge about the cached image right now so I currently can't really work on it. I will try to understand it more later on the project, and will surely look into this issue right after. I understand that the same image that counts more is annoying the sooner. As far as I think, I agree, the downloaded songs should be "untouchable" and therefore shouldn't be deleted.

> yeah and it would be cool to upload custom images to fix normal video images to be 16:9 rectangles insteal of a 1:1 square

I don't really understand the idea behind this, if you want to upload an image to replace the current one of the songs, it's clearly out of scope and we demand too much work right now

**Comment by fanieko** [comment]
Hi
> I don't really understand the idea behind this, if you want to upload an image to replace the current one of the songs, it's clearly out of scope and we demand too much work right now

I just wanted this bcs not every song is in the default YouTube music format and sometimes it is a normal video, the problem is that the image is a 16:9 rectangle. Maybe try croping the image? 

I was just asking if you could fix that bcs it doenst feel right, but it can be dificult to implent this.

Thanks for mantaining this amazing app, sadly i am not that skilled and can't create a fork and do it myself, so sorry if i am demaning to much stuff from you.

**Comment by Malopieds** [comment]
> I just wanted this bcs not every song is in the default YouTube music format and sometimes it is a normal video, the problem is that the image is a 16:9 rectangle. Maybe try croping the image?

Well, I believe that we should be able to differentiate the songs and the videos, that's why the 16:9 aspect ratio, what do you think ?

**Comment by fanieko** [comment]
> Well, I believe that we should be able to differentiate the songs and the videos, that's why the 16:9 aspect ratio, what do you think ?

Yeah that makes sense, i think it could be necessary to some people after all.
What i don't like it that some videos that are 1:1 it crops to fit in the rectangle, but no problem, i was just sugesting :)
